docs/graphcast/radios/graphcast-cli.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 3
+---
+
+# Graphcast CLI
+
+The source code for the Graphcast CLI is available [on GitHub](https://github.com/graphops/graphcast-cli).
+
+## Introduction
+
+The Graphcast CLI enables sending one-off messages. Currently, its default behaviour is to construct and send a message of type `UpgradeIntentMessage`, which is used for the [Subgraph Upgrade Pre-sync feature](https://docs.graphops.xyz/graphcast/design-principles#subgraph-upgrade-pre-sync) of Subgraph Radio.
```

Comment:
There is no default behavior anymore, subcommand is required.

It would be good to explain how the 2 current subcmd fit together for the presyncing feature